### ISSUE TITLE
Background color on hover for inline notification does not meet AA contrast

### ISSUE THREAD

**Issue by joshblack** [carbon-design-system/carbon#3536]
When hovering over the action, it seems like the text may not be hitting AA contrast:

![image](https://user-images.githubusercontent.com/3901764/61843821-ce23ee00-ae62-11e9-9276-a88e0f6e9da7.png)

**Comment by dakahn** [comment]
closing in favor of #3715
